src/backend/utils/adt/arrayfuncs.c

```diff
@@ -945,6 +945,8 @@ CopyArrayEls(ArrayType *array,
 	int			bitmask = 1;
 	int			i;
 
+	Assert(values || nitems == 0);
```

Comment:
I think there is no reason to call CopyArrayEls with NULL as values and 0 as nitems

Reply:
Removed == 0 condition and switched to Insist